inst/extdata/chromosome_arms_grch37.tsv

```diff
@@ -0,0 +1,49 @@
+chromosome	start	end	arm
+1	10000	121500000	p
+1	142600000	249250621	q
```

Comment:
Handy! I was getting these from a package for another function but can switch that code to use this directly